content/post/2026/02/i-failed-but-feel-good.md

```
@@ -0,0 +1,135 @@
+
+I refreshed my website... and it didn't show.
+
+Turns out...I'd forgotten to switch it from draft... whoops!
+
+## Accepting 
+
+Technically... TECHICALLY! I did _blog_ three times. Just one of them was a draft..
+
+But lets be real, that would be an lettter-of-the-law thing. And honestly, I wasn't proud of the post. It was unfocused and rambling and clearly done in a rush. I'm not happy with that and it'd leave a sour taste about blogging for the future.
+
+So lets nuke that (feel free to go read the post from the commit if you want)[LINK TO COMMIT] and replace it with something better.
```

markdownlint-cli2 (0.21.0)

[warning] Reversed link syntax
(feel free to go read the post from the commit if you want)